Add known issue - unavailable vmdp (#460)
harvester/docs: versioned_docs/version-v1.2/vm/create-windows-vm.md

3. The **third volume** is a `Container` with the following values:
    1. `Name`: The value `virtio-container-disk` is set by default. You can keep it or change it.
    2. `Type`: Select `cd-rom`.

>     3. `Docker Image`: The value `registry.suse.com/suse/vmdp/vmdp:2.5.4.2` is set by default. It's recommended you don't change it.

Reviewer: ```suggestion
    3. `Docker Image`: The value `registry.suse.com/suse/vmdp/vmdp:2.5.4.2` is set by default. We recommend not changing this value. 
```

Author: Updated it. Thank you.

>     4. `Bus`: The value `SATA` is set by default. It's recommended you don't change it.

Reviewer: ```suggestion
    4. `Bus`: The value `SATA` is set by default. We recommend not changing this value. 
```

Author: Updated it. Thank you.
